source/features/add-co-authored-by.tsx

```diff
@@ -37,7 +37,7 @@ async function fetchCoAuthoredData() {
 }
 
 function addCoAuthors() {
-	const field = select('#merge_message_field');
+	const field = select<HTMLInputElement>('#merge_message_field');
```

Comment:
So as mentioned elsewhere, the generic default value is `HTMLElement`, but in this case we need it as an `HTMLInputElement` so that we can access the `value` property.